email_reply_parser/__init__.py

```diff
@@ -215,7 +215,7 @@ def read(self):
 
         self.found_visible = False
 
-        is_multi_quote_header = self.MULTI_QUOTE_HDR_REGEX.search(self.text)
+        is_multi_quote_header = self.MULTI_QUOTE_HDR_REGEX.search(self.text.strip())
```

Comment:
Any chance `text` is `None` or something? Have seen that happen before in crazy circumstances.

Reply:
It should not although we should handle that more generally in this library (but not tonight)